gateway-js/package.json

```diff
@@ -28,9 +28,10 @@
     "@apollo/core-schema": "^0.2.0",
     "@apollo/federation": "file:../federation-js",
     "@apollo/query-planner": "file:../query-planner-js",
-    "@apollo/server-gateway-interface": "^1.0.2",
+    "@apollo/usage-reporting-protobuf": "^4.0.2",
```

Comment:
The `@apollo/server-gateway-interface` package actually requires node 14 so I removed it and copied the code into the repo instead. I don't think we'll be making any changes to the interface that affect v0 so this should be ok.